### PR TITLE
Add connectable beacons article

### PR DESCRIPTION
**Pull request by beaufortfrancois**
WIP

### CODE REVIEW

File: src/content/en/updates/posts/2016/04/connect-to-physical-web-beacons.markdown [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+TODO
+
+## Connectable Physical Web Beacon
+
+TODO
+
+### Using Node.js
+
+The [node-eddystone-beacon](https://github.com/don/node-eddystone-beacon) is a
+Node.js module for creating Eddystone beacons. It uses under the hood the
```
Comment (jpmedley) [inline src/content/en/updates/posts/2016/04/connect-to-physical-web-beacons.markdown]: "It uses under the hood" --> "Under the hood it uses"

**Comment by jpmedley** [comment]
@petele @jeffposnick 

This is light on text and heavy on code. Would this be better as a Chrome sample than as an /updates article?

**Comment by jeffposnick** [comment]
Given that it's tied to some third-party libraries, I don't know that it's appropriate for https://github.com/GoogleChrome/samples

If it's a project that ties a bunch of libraries together to build a cool, standalone project, then perhaps the code should live in its own dedicated repo, and if there's anything written for /web/updates/, it's a summary of the project and a link to that repo?

File: src/content/en/updates/posts/2016/04/connect-to-physical-web-beacons.markdown (hunk 1 above)
Comment (beaufortfrancois) [inline src/content/en/updates/posts/2016/04/connect-to-physical-web-beacons.markdown]: Thank you!

Comment (beaufortfrancois) [inline src/content/en/updates/posts/2016/04/connect-to-physical-web-beacons.markdown]: Thank you!

**Comment by beaufortfrancois** [comment]
Sorry for not being explicit there. This is just `me` doing a `git push` while I'm accumulating resources on this. There will be more text eventually.
I didn't consider https://github.com/GoogleChrome/samples because as @jeffposnick mentioned, this is more of a collection of recipes/snippets for developers to get a physical web beacon connectable and usable with Web Bluetooth.

I'll keep working on this and will ask for review when time comes.

Thanks for having a look early!

**Comment by beaufortfrancois** [comment]
Sorry for not being explicit there. This is just `me` doing a `git push` while I'm accumulating resources on this. There will be more text eventually.
I didn't consider https://github.com/GoogleChrome/samples because as @jeffposnick mentioned, this is more of a collection of recipes/snippets for developers to get a physical web beacon connectable and usable with Web Bluetooth.

I'll keep working on this and will ask for review when time comes.

Thanks for having a look early!

**Comment by petele** [comment]
hey @beaufortfrancois - any updates on this? happy to keep it in purgatory if necessary.

**Comment by beaufortfrancois** [comment]
I'd like to keep this one opened fi that's OK with you.

**Comment by petele** [comment]
Just the regular check in. ;) Happy to keep it open.

**Comment by beaufortfrancois** [comment]
Opened yes ;)

**Comment by petele** [comment]
Assign to me when it's ready to go.

**Comment by beaufortfrancois** [comment]
Closing this PR so that I can clean this branch.
I've kept a copy of this PR at https://github.com/beaufortfrancois/WebFundamentals/commit/2be1fdd021910087821df72397034cf50044d8cc

**Pull request closed without merging**